Sources/FluentDarkModeKit/Extensions/UILabel+DarkModeKit.swift

```diff
@@ -9,8 +9,18 @@ extension UILabel {
   }
 
   static let swizzleDidMoveToWindowOnce: Void = {
```

Comment:
UILabel does not really override didMoveToWindow from UIView. Why do we need to swizzle UILabel's `didMoveToWindow` here?

Reply:
I don't remember. cc @kylinroc